style.css

```diff
@@ -42,23 +42,52 @@ main {
 button {
     padding: 0 1vw;
     margin: 1vw;
-    height: 50px;
-    background-color: #FFFFFF;
+    min-height: 50px;
     background-size: contain;
     background-repeat: no-repeat;
     background-position: center;
 }
 
 .painter {
-    background-image: url('paintbrush.png');
+    background-color: #000000;
 }
 
 .eraser {
     background-image: url('eraser.png');
+    background-color: #FFFFFF;
+}
+.red {
+    background-color: #ff0000;
+}
+
+.blue {
+    background-color: #0000FF;
+}
+
+.green {
+    background-color: #008000;
+}
+
+.yellow {
+    background-color: #ffff00;
+}
+
+.orange {
+    background-color: #ffa500;
 }
 
 h1 {
-    font-family: "Hiragino Maru Gothic Pro";
+    font-family: "Hiragino Maru Gothic Pro", sans-serif;
     color: #FFFFFF;
     overflow: hidden;
 }
+
+.clicked {
```

Comment:
This could target '.painter' as well to make it more readable

Reply:
Painter has now been renamed to .black